src/sass/extends/_sidebar.scss

```diff
@@ -21,35 +21,78 @@ Sidebar Navigation
 2. Sidebar
 --------------------------------------------------------------------*/
 
+.sidebar-toggler {
+  border: 1px solid $uds-color-base-gray-3;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: $uds-size-spacing-3 $uds-size-spacing-2;
+
+  p {
+    margin: 0;
+  }
+
+  svg {
+    transition: all 0.3s;
+  }
+
+  &[aria-expanded='false'] {
+    svg {
+      transform: rotate(-180deg);
+    }
+  }
+
+  @include media-breakpoint-up(md) {
+    display: none;
+  }
+  @include media-breakpoint-down(md) {
+    margin: 0 $uds-size-spacing-4;
+  }
+}
+
```

Comment:
I can't discern what these changes might be related to and there is no working example within Storybook to explain why this option exists. Further investigation is warranted, but including this markup in our project shouldn't pose a challenge.

Reply:
The `sidebar-toggler` appears to be the mobile presentation of a sidebar menu, as shown here from Storybook:

<img width="526" alt="Screen Shot 2021-08-04 at 3 52 38 PM" src="https://user-images.githubusercontent.com/31013359/128265580-80782702-4ea6-4e83-9043-68fd800265df.png">

It's a `<div>` that sits above the actual menu items and has that 'Select Section' link for toggling the menu visibility. I made a note in Github to discuss this on Friday. We don't appear to be using the sidebar markup from Storybook.